Add a banner to send people to the latest key rotation docs (#12303)
DataDog/documentation: content/en/agent/faq/rpm-gpg-key-rotation-agent-6.md

kind: faq
---

<div class="alert alert-warning">

> This page refers to the key rotation that happened in 2019, there's a separate documentation page for the <a href="agent/faq/linux-agent-2022-key-rotation">2022 Key Rotation</a>.

Reviewer: ```suggestion
This page pertains to the 2019 key rotation. For the 2022 key rotation, consult the <a href="agent/faq/linux-agent-2022-key-rotation">2022 Linux Agent Key Rotation</a> documentation.
```
